### ISSUE TITLE
Cropping with LOS PM

### ISSUE THREAD

**Issue by diegotibi** [Cloud-RF/Signal-Server#27]
Not an issue, more a feature request.

Is it possible to crop the image when propagation model is set to LOS (2)

Thanks

**Comment by FarrantAlex** [comment]
Fair point. The LOS routine is very different to the models for historic reasons and needs refactoring.
A bonus of refactoring would be you could style the LOS instead of just green. I've added a comment at the right point to steer someone but don't have the time to refactor this.
